### ISSUE TITLE
Create a One Sheet: Public Tree Map

### ISSUE THREAD

**Issue by Oluwafemi-Jegede** [hackforla/product-management#58]
### Overview
Create the project one sheet so it can be shared with stakeholders and funders.

### Action Items
- [ ] Check to confirm project does not already have a one sheet
- [ ] Speak to the team to get an overview of the project
- [ ] Include project logo
- [ ] Create history/last 6 months
- [ ] Create roadmap/next 6 months
- [ ] Include last updated date

### Resources/Instructions
Check issue #3 for sample one sheets

**Comment by ExperimentsInHonesty** [comment]
@Oluwafemi-Jegede This project is active but might be a little slow to respond.  The project lead is Emily who is super passionate about the topic and has done amazing work.  She may or may not be enthusiastic about participating in making this one sheet.  So it might be more of an interview process then come back with information we can craft into a one sheet.

**Comment by Oluwafemi-Jegede** [comment]
@ExperimentsInHonesty Okay noted. 
Let me know when we can have a comprehensive disccusion on the project

Thanks

**Comment by ExperimentsInHonesty** [comment]
@Oluwafemi-Jegede I can talk to you after you have reached out to Emily.  If you are still unclear about what is required, please reach out to Olivia for guidance.

**Comment by ExperimentsInHonesty** [comment]
@Oluwafemi-Jegede Have you heard from @emillipede ?

**Comment by Oluwafemi-Jegede** [comment]
@ExperimentsInHonesty I am yet to hear from @emillipede , I have gone ahead to reach out to @Olivia-Chiong for help as advised

**Comment by ExperimentsInHonesty** [comment]
I didn't hear from emily yet.  We will change the status of the project to completed and remove the looking for from the project card if I don't hear from her in two weeks from the day I last wrote. In the meantime we will put this issue on hold

**Comment by ExperimentsInHonesty** [comment]
@Oluwafemi-Jegede - Emily is MIA.  We will have to write the one sheet without her as a completed project.  Let's talk about this at the next pm meeting you attend.

**Comment by Oluwafemi-Jegede** [comment]
Okay this is noted, I will be available for the Friday meeting

**Comment by Oluwafemi-Jegede** [comment]
Discussion with @ExperimentsInHonesty 

Things to take note of when preparing the One-Sheet 

1. Document to be in past tense has project has been completed
2. Explore scraping information from Test website (about and data sources section) and Instagram page
3. Research on Trees e.g  Santa Monica and add to one-sheet
4. Add pictures to one-sheet and sources 
5. Prepare the first draft and share for review by next meeting

**Comment by Olivia-Chiong** [comment]
@Oluwafemi-Jegede 
Please provide an update before 2/23 meeting.
Progress:
Blocks:
Availability:
ETA:

**Comment by Oluwafemi-Jegede** [comment]
@Olivia-Chiong I will not be joining today's meeting because of time zone differences, I will attend the Friday meeting to give a better update. 

Progress: Data gathering ongoing
Blocks: Some personal events delayed the delivery, last Friday
Availability: Friday, 26th
ETA: Friday, 26th

**Comment by Oluwafemi-Jegede** [comment]
@Olivia-Chiong One sheet Updated.

Please let me know if you have questions.

NB: Project road map converted to project milestones.

**Comment by ExperimentsInHonesty** [comment]
I made some updates to the one sheet.  Please reach out to Ben Swerdlow on slack to see if he has any other items to add to the technical milestones.  Or if he can recommend we talk to anyone else besides Emily who is not responding.

https://docs.google.com/document/d/1paiAKHTot8YmKgAayNC3jjg_rRblwIOXR2sM4WVczqU/edit

**Comment by ExperimentsInHonesty** [comment]
Completed one sheet.  Needs to be made into pdf
